packages/ripple-nuxt-tide/lib/layouts/default.vue

```diff
@@ -169,6 +170,19 @@ export default {
     // Set ripple quickexit option by tide site settings
     this.rplOptions.quickexit = this.$store.state.tide.siteData.field_site_show_exit_site
     this.rplOptions.origin = this.$store.state.tide.protocol + '//' + this.$store.state.tide.host
+    this.rplOptions.imgQueryString = (bp) => {
```

Comment:
Instead of define this function in layout, should we move it plugin(https://github.com/dpc-sdp/ripple/blob/master/packages/ripple-nuxt-ui/lib/templates/plugin.js) as a default and let it able to be overridden? To support things outside kraken.